docs/html/reference/pip_show.rst

```diff
@@ -25,10 +25,29 @@ Options
 .. pip-command-options:: show
 
 
+Format
+======
+
+The various fields present in the JSON output and their explanation is as follows.
+
+- **metadata:** A dictionary with the core metadata fields present in the METADATA file, as defined in the
+  `Core metadata specifications`_. The fields are lower cased, with dashes replaced by underscores.
+- **direct_url:** A dictionary containing the content of direct_url.json, if present, as specified in `PEP 610`_
+- **installer:** A string containing the content of INSTALLER, if present, as specified in `PEP 376`_
+- **record:** A list of {file_path, file_content_hash, file_size|, representing the content of RECORD,
+  if present, as specified in `PEP 376`_ .
+- **requested:** A boolean, set to True if REQUESTED is present, as specified in `PEP 376`_
+- **required_by:** A list of canonicalized distribution names that depend on the distribution in question
+- **location:** A string containing the path where the distribution is installed.
```

Comment:
Hi, I think [definition list markup](https://docutils.sourceforge.io/docs/user/rst/quickref.html#definition-lists) might fit this part better.

Reply:
Hi @McSinyx 

Did you mean something like this?

```
- **metadata:**

    A dictionary with the core metadata fields present in the METADATA file, as defined in the
    `Core metadata specifications`_. The fields are lower cased, with dashes replaced by underscores.

- **direct_url:**

    A dictionary containing the content of direct_url.json, if present, as specified in `PEP 610`_
```

Reply:
I meant something line this
<details><summary>Restructured Text</summary><p>

```rst
metadata
   A dictionary with the core metadata fields present in the ``METADATA`` file,
   as defined in the `Core metadata specifications`_. The fields are
   lower cased, with dashes replaced by underscores.

direct_url
   A dictionary containing the content of ``direct_url.json``,
   if present, as specified in :pep`610`.

installer
   A string containing the content of ``INSTALLER``,
   if present, as specified in :pep:`376`.

record
   A list of ``[file_path, file_content_hash, file_size]``, representing
   the content of ``RECORD``, if present, as specified in :pep:`376`.

requested
   A boolean, set to True if ``REQUESTED`` is present,
   as specified in :pep:`376`.

required_by
   A list of canonicalized distribution names that depend
   on the queried distribution.

location
   A string containing the path where the distribution is installed.

.. _`Core metadata specifications`: https://packaging.python.org/specifications/core-metadata/
```

</details></p>
which renders
<details><summary>Screenshot</summary><p>

![2020-04-10T15:41:54](https://user-images.githubusercontent.com/13689192/78977150-d54e7180-7b41-11ea-98f8-441540c573d7.png)

</details></p>

I nit a few spots, e.g. typos and PEP references, though I have no strong opinion on the bold part.

Reply:
Thanks @McSinyx ,

That looks much more readable than what I had. . I have italized the fieldnames instead of making them bold. Also TIL that ``` :pep:`376`. ``` automatically creates the PEP link.

Reply:
Hi @sbidoul 

I have reformatted the content according to the suggestions with some help from @McSinyx . Please take a look at the same and let me know if further changes are needed